server/src/main/java/io/druid/server/log/LoggingRequestLogger.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.server.log;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import com.metamx.common.logger.Logger;
+import io.druid.query.Query;
+import io.druid.server.RequestLogLine;
+import org.slf4j.MDC;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class LoggingRequestLogger implements RequestLogger
+{
+  final ObjectMapper mapper;
+  final boolean setMDC;
+  final boolean setContextMDC;
+
+  public LoggingRequestLogger(
+      ObjectMapper mapper,
+      boolean setMDC,
+      boolean setContextMDC
+  )
+  {
+    this.mapper = mapper;
+    this.setMDC = setMDC;
+    this.setContextMDC = setContextMDC;
+  }
+
+  private static final Logger LOG = new Logger(LoggingRequestLogger.class);
```

Comment:
This class and the level (info) should be documented so people know where they're going to appear. One nice thing this enables is using a different appender for the request logs based on knowing they're going to appear under `io.druid.server.log.LoggingRequestLogger`.